### PR TITLE
Add license and manifest

### PR DESCRIPTION
**Pull request by mrocklin**
cc @jakirkham your expertise would be welcome here.  Anything we're missing?  If you have a chance to glance at the setup.py that would probably also be a good idea.

### CODE REVIEW

**Comment by mrocklin** [comment]
Once this goes in I'm pretty much ready to publish and point a bunch of our projects to start using this immediately.  Any objections?

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+recursive-include dask_sphinx_theme *.py
+recursive-include dask_sphinx_theme *.html
+recursive-include dask_sphinx_theme *.css
+recursive-include dask_sphinx_theme *.svg
+recursive-include dask_sphinx_theme *.conf
+
+include setup.py
```
Comment (jakirkham) [inline MANIFEST.in]: Looks like we need `requirements.txt` too as `setup.py` is loading that.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+include README.rst
+include LICENSE.txt
+include MANIFEST.in
```
Comment (jakirkham) [inline MANIFEST.in]: This gets added automatically. So doesn't need to be explicitly listed. Though it shouldn't hurt to have it.

File: MANIFEST.in (hunk 1 above)
Comment (jakirkham) [reply MANIFEST.in]: Note: Done below.

**Comment by jakirkham** [comment]
LGTM. Thanks Matt 😄

**Comment by mrocklin** [comment]
Thanks for the rapid review

**Pull request merged**